app/assets/javascripts/app/views/help_view.js

```diff
@@ -133,6 +136,32 @@ app.views.Help = app.views.StaticContentView.extend({
     this.$('#faq').append(help_section.render().el);
   },
 
+  /**
+   * Returns the jQuery <a> element whose data-section property equals the given query
```

Comment:
Sure, it needs code review. In fact the most appropriate would be `findElByDataSection` as it is only based on the `data-section` property.
I'm really not comfortable with JS so numerous things might need to be changed.